src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/LightUtilitySpell.java

```diff
@@ -135,12 +135,16 @@ public void refresh(ModConfigEvent event) {
     @OnlyIn(Dist.CLIENT)
     @Mod.EventBusSubscriber(modid = MineBound.MOD_ID, value = {Dist.CLIENT})
     public static class GlowingRenderer implements ResourceManagerReloadListener {
-        protected static Field field = null;
+        protected static Field effectField = null;
         
         static {
             try {
-                field = LevelRenderer.class.getDeclaredField("entityEffect");
-                field.setAccessible(true);
+                var fields = LevelRenderer.class.getDeclaredFields();
+                for (Field field : fields) {
+                    if(field.getType().equals(PostChain.class))
+                        effectField = field;
+                }
+                effectField.setAccessible(true);
```

Comment:
Looks like it's getting the wrong object here. It's getting `transparencyChain`. I think you'll just need a `break` inside the if statement.

Reply:
Oh, I thought I pushed it. I have that it my local; I was testing on the actual game instead of the dev version